src/GraphMetadataViewExtension/GraphMetadataView.xaml

```diff
@@ -0,0 +1,194 @@
+                                       Style="{StaticResource headerStyle}" />
+
+                        </DataTemplate>
+                    </Setter.Value>
+                </Setter>
+            </Style>
+
+        </ResourceDictionary>
+    </UserControl.Resources>
+    
+    <Grid Margin="10px 10px">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <StackPanel Orientation="Vertical"
+                    Grid.Row="0">
+            <GroupBox Header="Description"
+                      Height="100">
+                <TextBox Text="{Binding GraphDescription, Mode=TwoWay, FallbackValue='Graph Description'}" 
+                         Style="{StaticResource TextBoxStyle}"/>
+            </GroupBox>
+            
+            <GroupBox Header="Image">
+                <controls:ImageSelector MaxHeight="150"
+                                        MinHeight="100"
+                                        Image="{Binding Thumbnail, Mode=TwoWay}"/>
+            </GroupBox>
+            
+            <GroupBox Header="Author Name">
+                <TextBox Text="{Binding GraphAuthor, Mode=TwoWay}"
+                         Style="{StaticResource TextBoxStyle}" />
+            </GroupBox>
+            
+            <GroupBox Header="Learn more URL">
+                <TextBox Validation.ErrorTemplate="{StaticResource ResourceKey=textboxErrorTemplate}"
+                         Name="UriTextBox"
+                         Style="{StaticResource TextBoxStyle}">
+                    <TextBox.Text>
+                        <Binding Path="HelpLink"
+                                 UpdateSourceTrigger="PropertyChanged"
+                                 ValidatesOnDataErrors="True"
+                                 Mode="TwoWay">
+                            <Binding.ValidationRules>
+                                <vr:UriValidationRule ValidatesOnTargetUpdated="True" />
+                            </Binding.ValidationRules>
+                            <Binding.Converter>
+                                <convertes:UriToStringConverter />
+                            </Binding.Converter>
+                        </Binding>
+                    </TextBox.Text>
+                </TextBox>
+            </GroupBox>
+            
+            <Separator Background="{StaticResource FilterIconColor}" Margin="0,20,0,20"/>
+        </StackPanel>
+
+        <Grid Grid.Row="1">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="*" />
+            </Grid.RowDefinitions>
+            <Grid Grid.Row="0"
+                  Margin="10px, 5px">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="20" />
+                </Grid.ColumnDefinitions>
+                <TextBlock Grid.Column="0"
```

Comment:
Can we activate tabbing ? It would be nice to have when dealing with multiple properties